content/id/docs/tasks/administer-cluster/manage-resources/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Mengelola Memori, CPU, dan Sumber Daya API
```

Comment:
It is also a good point! We need an index for "Mengelola Memori, CPU, dan Sumber Daya API"